internal/testutil/treedump.go

```diff
@@ -55,11 +58,32 @@ func TreeDump(dir string) map[string]string {
 		default:
 			return fmt.Errorf("unknown file type %d: %s", ftype, fpath)
 		}
+
+		if ftype != fs.ModeDir {
+			stat, ok := finfo.Sys().(*syscall.Stat_t)
+			if !ok {
+				return fmt.Errorf("cannot get syscall stat info for %q", path)
+			}
+			inode := stat.Ino
+			if len(pathsByInodes[inode]) == 1 {
+				inodes = append(inodes, inode)
+			}
+			pathsByInodes[inode] = append(pathsByInodes[inode], "/"+path)
```

Comment:
"/"+path here can be error prone if we change it later. Can you refactor the `path` above so it is updated in all the branches like in the suggestion I sent you? This way if we change the paths above it will be automatically valid here as well.

Reply:
I see this is changed, but changing the `path` var would change this error a bit as well:

```go
			stat, ok := finfo.Sys().(*syscall.Stat_t)
			if !ok {
				return fmt.Errorf("cannot get syscall stat info for %q", path)
			}
```

Suddenly you will start getting absolute paths, which is probably fine but not consistent with the error above:

```go
		finfo, err := d.Info()
		if err != nil {
			return fmt.Errorf("cannot get stat info for %q: %w", path, err)
		}
```

Reply:
Agreed offline: let's use `fpath` for all of these errors.